webapp/src/pages/eventConfig/components/EventConfigComponent.js

```diff
@@ -57,9 +87,113 @@ class EventConfigComponent extends Component {
         isEdited = true;
       }
     }
+    this.validateEventDetails();
     this.setState({
       hasBeenUpdated: isEdited
     });
+
+  };
+
+  getErrorMessages = errors => {
+    let errorMessages = [];
+
+    for (let i = 0; i < errors.length; i++) {
+      errorMessages.push( //TODO: warning appears for this - each child in list should have key prop
```

Comment:
The key needs to be unique to distinguish items in a list, so you could do something like `key={"error" + i}`

Reply:
Good to know. Thanks.